### PR TITLE
Pod priority.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: HostAliases
+id: HostAliases
+date: 2019-01-31
+full_link: /docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core
+short_description: >
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/host-aliases.md]:
```
does `>` exist in the source file?
```

File: content/zh/docs/reference/glossary/pod-priority.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Pod Priority
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/pod-priority.md]:
```suggestion
title: Pod 优先级
```

File: content/zh/docs/reference/glossary/pod-priority.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+id: pod-priority
+date: 2019-01-31
+full_link: /docs/concepts/configuration/pod-priority-preemption/#pod-priority
+short_description: >
+  Pod Priority indicates the importance of a Pod relative to other Pods.
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/pod-priority.md]:
```suggestion
  Pod 优先级表示 Pod 相对于其他 Pod 的重要性。
```

File: content/zh/docs/reference/glossary/pod-priority.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+aka:
+tags:
+- operation
+---
+<!--
+ Pod Priority indicates the importance of a Pod relative to other Pods.
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/pod-priority.md]: delete this

File: content/zh/docs/reference/glossary/pod-priority.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+-->
+
+
+Pod 优先级表示 Pod 相对于其他 Pod 的重要性。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/pod-priority.md]: delete this

File: content/zh/docs/reference/glossary/pod-priority.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+<!--more-->
+
+<!--
+[Pod Priority](/docs/concepts/configuration/pod-priority-preemption/#pod-priority) gives the ability to set scheduling priority of a Pod to be higher and lower than other Pods — an important feature for production clusters workload.
+-->
+
+[Pod Priority](/docs/concepts/configuration/pod-priority-preemption/#pod-priority) 使Pod能够将调度优先级设置为高于和低于其他Pod  - 这是生产集群工作负载的一项重要功能。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/pod-priority.md]:
```suggestion
[Pod 优先级](/docs/concepts/configuration/pod-priority-preemption/#pod-priority) 使 Pod 能够将调度优先级设置为高于和低于其他 Pod  - 这是生产集群工作负载的一项重要功能。
```

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  A HostAliases is a mapping between the IP address and hostname to be injected into a Pod's hosts file.
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/host-aliases.md]:
```suggestion
  HostAliases 是要注入 Pod 的 hosts 文件的 IP 地址和主机名之间的映射。
```

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+aka:
+tags:
+- operation
+---
+<!--
+ A HostAliases is a mapping between the IP address and hostname to be injected into a Pod's hosts file.
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/host-aliases.md]: delete this

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+-->
+
+HostAliases 是要注入 Pod 的 hosts 文件的 IP 地址和主机名之间的映射。
```
Comment (xichengliudui) [inline content/zh/docs/reference/glossary/host-aliases.md]: delete this

**Comment by xichengliudui** [comment]
@congfairy ,thanks 
The basic branch of Chinese translation has been updated. Please do not work on 1.14

**Comment by tengqm** [comment]
@xichengliudui I'm not sure the Chinese translation in 1.14 has been merged back to master or release-1.15.

**Comment by xichengliudui** [comment]
@tengqm 1.14 based translations have been merged into the 1.16 branch.
@congfairy plase create a new pr based on the 1.16 branch, ref: https://github.com/k8smeetup/k8s-official-translation/blob/master/README.md#%E5%89%8D%E6%83%85%E6%A6%82%E8%A6%81

**Comment by xichengliudui** [comment]
It's also merged into the master https://github.com/kubernetes/website/pull/16584

**Pull request closed without merging**